app/src/main/res/values-pl/Main.xml

```diff
@@ -152,4 +152,14 @@
   <string name="send_data_alert_title">Dziękujemy!</string>
   <!--Alert po wysłaniu danych-->
   <string name="send_data_alert_description">Twoje dane zostały przekazane do Głównego Inspektoratu Sanitarnego.\nZapewniamy, że żadne osoby, którym ewentualnie się skontaktujemy nie poznają Twoich danych osobowych.\n\nW razie jakichkolwiek pytań napisz do nas na %@\nlub zadzwoń %@</string>
+  <!--Nazwa kanału powiadomienia wskazującego na działanie usługi w tle-->
+  <string name="channel_name">Usługa bluetooth</string>
```

Comment:
Czy mógłbyś przenieść te teksty do strings.xml? Pliki Main są generowane automatycznie z osobnego zasobu, nie chciałbym stracić tych tekstów.

Reply:
Jasne, a ten proces jest gdzieś opisany? Warto by to zaznaczyć żeby nie powielać błędu, bo ja założyłem że taka jest po prostu założona struktura

Reply:
@jakublipinski ?

Reply:
Teksty generowane są z tego doc'a: https://docs.google.com/spreadsheets/d/1vpJiu2jJcxBFWefIyi__QZZH6nwsIVktPFHEohJNMsQ/edit#gid=0 przy pomocy skryptu: https://github.com/jakublipinski/export-gsheet-to-app-resources

Opiszę to jutro gdzieś na repo. Sorki za zamieszanie @maciekjanusz . Jak potrzebujesz jakieś nowe teksty to najlepiej zgłoś w jakimś issue i oznacz mnie.